src/test/java/nextstep/ladder/domain/ParticipantsTest.java

```diff
@@ -4,22 +4,42 @@
 import org.junit.jupiter.api.Test;
 
 import java.util.List;
+import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class ParticipantsTest {
     @Test
-    @DisplayName("2가지의 생성자 동작 확인")
+    @DisplayName("기본 생성자와 팩토리 메소드 생성 동작 일치 여부 확인")
     void create() {
-        assertThat(new Participants(new String[]{"p1", "p2"})).isEqualTo(new Participants(List.of("p1", "p2")));
+        assertThat(Participants.fromNames(List.of(new Name("p1"), new Name("p2"))))
+                .isEqualTo(new Participants(List.of(new Participant("p1", 0), new Participant("p2", 1))));
     }
 
     @Test
     @DisplayName("참여자가 2명 미만인 경우 테스트 - 예외 발생")
     void invalid_name_size() {
-        assertThatThrownBy(() -> new Participants(new String[]{"p123"}))
+        assertThatThrownBy(() -> new Participants(List.of(new Participant("p1"))))
```

Comment:
테스트 대상이 되는 메서드만으로 예외를 검증하면 어떨까요?
```suggestion
        List participants = List.of(new Participant("p1"));
        assertThatThrownBy(() -> new Participants(participants))
```